Prace_domowe/Praca_domowa1/Grupa4/Hubert_Drążkowski/Hubert_Drazkowski_PD2.ipynb

```diff
@@ -0,0 +1,2196 @@
+{
```

Comment:
<p>Będziemy baczyć, a jakieś wskazówki jak radzić sobie z niesnaskami owego kapryśnego Target Encodera?</p><p> </p><p>Jak można podejrzeć w dokumentacji, to chociaż link być podał \:p</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>